Makefile

```diff
@@ -1,32 +1,137 @@
-.PHONY: all build
+# Configure environment constants
+ACCOUNT_KEYS_FILENAME = "./account-keys.csv"
+FLOW_JSON = script/flow.json
+FLOW_JSON_NETWORK = localnet
+FLOW_JSON_SIGNER = localnet-service-account
+FLOW_CLI_FLAGS = -n $(FLOW_JSON_NETWORK) -f $(FLOW_JSON) --signer $(FLOW_JSON_SIGNER)
+ROSETTA_ENV = localnet
+ROSETTA_HOST_URL = "http://127.0.0.1:8080"
+COMPILER_FLAGS := CGO_CFLAGS="-O2 -D__BLST_PORTABLE__"
 
+.PHONY: all build
 all: build
 
+.PHONY: go-build
 go-build:
-	go build -o server cmd/server/server.go
+	${COMPILER_FLAGS} go build -o server cmd/server/server.go
+
+.PHONY: gen-originator-account
+gen-originator-account:
```

Comment:
what do you think about making these new methods small bash scripts instead?

Reply:
Maybe another time. I don't like bash scripts or make targets much but if someone wants to go to town in the next cycle and tidying things up more then great.